### PR TITLE
Add section "Internal pdf viewer" to manual

### PR DESCRIPTION
**Pull request by octaeder**
this includes description for changing pageOffset in subsection "Special features"

### CODE REVIEW

File: utilities/manual/usermanual_en.html [hunk 1]
```diff
@@ -568,6 +569,37 @@ <h2 id="SECTION23">3.2 The log files</h2>
...
+
+<p>You may want to change otions in the config dialog (s. Internal PDF Viewer).</p>
+<h3 id="SUBSECTION241">3.3.1 Special features</h3>
+<p>The viewer arranges the pages in a grid of columns and rows in which the pages are placed. Normally, the first page is
+placed on the left in the first row and the following pages fill the row and so on. This behavior may not be appropriate
+in all cases. For example, if you are working on a book, since books have odd pages on the right. To meet this
+requirement, the first page is automatically moved to the right when you select the 2x1 grid from the menu. The space on
+the left side remains empty. You can manually change the location for the first page in any grid. To do this, use Shift
++ left mouse button click on any place of the grid while in Scroll mode. The first page will be moved to the place that
+is in the first row and in the same column as the place you selected.</p>
```
Comment (egidioln) [inline utilities/manual/usermanual_en.html:602]:
```suggestion
is in the first row and in the same column as the place you selected.</p>
<p>You can also use the pdf viewer to get the (x,y) coordinates of a point by 
performing ctrl + shift + left mouse button click on a given page. By doing so, the 
(x,y) coordinates of the mouse position (in centimeters) with respect to the bottom 
left corner of the current page are copied into your clipboard. This becomes 
particularly useful when adjusting margins or working with TikZ.</p>
```

File: utilities/manual/usermanual_en.html [hunk 2]
```diff
@@ -568,6 +569,37 @@ <h2 id="SECTION23">3.2 The log files</h2>
...
+<ul>
+<li>Magnify</li>
+		<ul>
+			<li>left mouse button click opens magnifier showing enlarged part of the text</li>
+			<li>Shift + left mouse button click increase zoom level</li>
+			<li>Alt + left mouse button click decrease zoom level</li>
+		</ul>
+<li>Scroll</li>
+		<ul>
+			<li>left mouse button click graps the document so you can scroll it around or follows a link</li>
```
Comment (sunderme) [inline utilities/manual/usermanual_en.html:588]: graps -> grabs

Comment (octaeder) [reply utilities/manual/usermanual_en.html]: ah, there it is! thx

**Comment by octaeder** [comment]
@sunderme Please merge if ok

**Comment by sunderme** [comment]
thanks

**Pull request merged**